syntax="proto3";

import "Math/proto/math.proto";

package astre.ecs;

message VisualComponent 
{
    bool visible = 1;
    string vertex_buffer_name = 2;
    string shader_name = 3;
    astre.math.Vec4Serialized color = 4;
}